src/content/decentralized-identity.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Decentralized Identity 
+description: An overview of decentralized identity on Ethereum
+lang: en
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Centralized identity systems are fragmented, poorly secured, and offer minimal privacy protection
+summaryPoint2: Decentralized identity increases personal control of one's identity information.
+summaryPoint3: Decentralized ID projects built on Ethereum can revolutionize identity management. 
+---
+
+# What are identity, identifiers and attestations? {#what-are-identity-identifiers-attestations}
+
+## What is identity {#what-is-identity}
+
+Identity means an individual's sense of self, defined by a set of unique characteristics. Identity refers to the fact of being an _individual_, i.e., a distinct human entity. Identity could also refer to other non-human entities, such as an organization or authority.
+
+## What are identifiers? {#what-are-identifiers}
+
+An identifier is a piece of information that can be used to distinguish a person's identity. Common identifiers include the following: 
+- Name
+- Social security number/tax ID number 
+- Mobile number 
+- Date and place of birth
+- Digital identification credentials, e.g., passwords, email address, usernames
+
+## What are attestations? {#what-are-attestations}
+ 
+An attestation refers to a claim made by an entity about another entity. An example attestation is a driver's license issued by the Department of Motor Vehicles (DMV). The driver's license, which proves a specific claim (that the holder has legal permission to drive a car) is made by one entity (the DMV) about another entity (Bob, the driver). 
+
+Although attestations and identities are different, an attestation contains identifiers that point to an individual's identity. The driver's license mentioned earlier, for example, may have identifiers, like the holder’s name, date of birth, and nationality. 
+
+## What are decentralized identifiers? {#what-are-decentralized-identifiers}
+
+Decentralized identifiers (DIDs) are identifiers that are not issued, managed, or controlled by a central authority. Traditional identifiers, like an email address or mobile number, are reliant on intermediaries—email providers, mobile network operators, and so on. 
```

Comment:
As noted above, would add some of this content to the above section, and rephrase this a little:
```suggestion
As mentioned, traditional identifiers like legal name, email address or Twitter handle are all reliant on intermediaries such as a governments, email providers or social media platforms. Decentralized identifiers (DIDs) are identifiers that are not issued, managed, or controlled by any central authority.
```
""